build/Common.props

```diff
@@ -65,4 +65,12 @@
     <Compile Include="$(RepoRoot)\src\OpenTelemetry.Api\Internal\StatusHelper.cs" Link="Includes\StatusHelper.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IncludeCodeAnalysisAttributes)'=='true' Or '$(IncludeInstrumentationHelpers)'=='true' Or '$(IncludeDiagnosticSourceInstrumentationHelpers)'=='true'">
```

Comment:
Who sets `IncludeCodeAnalysisAttributes`? I'm not seeing it used.

Reply:
Currently no one. So maybe I should remove it - it was meant to be used if a project wants to include the attributes. But I guess this is typically done one by one here. What do you think?

Reply:
I kind of like the model of just including what you need. But others may have different opinions.

If no project is using the setting, I would drop it until someone uses it.